app/models/bridge.rb

```diff
@@ -43,4 +46,8 @@ class Bridge < ApplicationRecord
   def self.generate_inbound_url
```

Comment:
Why is this a class method? Do you need to generate a url outside of the class? I think this should be a private instance method.